mmcv/cnn/bricks/generalized_attention.py

```diff
@@ -131,7 +131,7 @@ def __init__(self,
 
             max_len_kv = int((max_len - 1.0) / self.kv_stride + 1)
             local_constraint_map = np.ones(
-                (max_len, max_len, max_len_kv, max_len_kv), dtype=np.int)
```

Comment:
might need to ensure the lowest version requirement of numpy for this cleaning

Reply:
I think it is safe because torch 1.3.0 corresponds to NumPy 1.21.2. However, `np.int` is a deprecated alias for the builtin `int` in NumPy 1.20.